exonum/src/blockchain/config.rs

```diff
@@ -79,17 +86,11 @@ impl ConsensusConfig {
     /// Checks if propose timeout is less than round timeout. Warns if fails.
     #[doc(hidden)]
     pub fn validate_configuration(&self) {
```

Comment:
is "validate" apropriate prefix, for function that only warns about invalid config?

Reply:
I'm not sure. :upside_down_face: What name do you propose? Is something like `check_configuration` better?

Reply:
I will fix it in a separate pull request. :upside_down_face: